src/test/groovy/graphql/execution/instrumentation/dataloader/Issue1178DataLoaderDispatchTest.groovy

```diff
@@ -79,11 +80,10 @@ class Issue1178DataLoaderDispatchTest extends Specification {
                 .build()
 
         then: "execution shouldn't error"
-        for (int i = 0; i < NUM_OF_REPS; i++) {
```

Comment:
why did this change ????

Reply:
Because I found that the     `@RepeatUntilFailure` is better way to achieve the same goal.